src/UnigramWordPredictor.java

```diff
@@ -51,7 +57,46 @@ public UnigramWordPredictor(Tokenizer tokenizer) {
   public void train(Scanner scanner) {
     List<String> trainingWords = tokenizer.tokenize(scanner);
 
+    // tokenizer is already breaking it down into individual tokens
+
+    // what i need to do is to put the individual token thats already in the trainwWords 
+    // list into an array/data structure of its own that will then 
+    // usr that token as a key and store and array as the values for that key for each word 
+    // that follows that token 
+    Map<String, List<String>> predictorMap = new HashMap<>();
+    
+    for(int i = 0; i < trainingWords.size() - 1; i++){
+      // grabbing the token to check 
+      String token = trainingWords.get(i);
+      // grabbing the word following the token to check
+      
+      // next time I need to ensure I am incrementing correct.../
+      String nextToken = trainingWords.get(i + 1);
+
+      if(predictorMap.containsKey(token)){
+        // if the new map were creating contains the token as a Key already
+        // then it will just add that nextToken to the list/array of strings associated
+        // with that token 
+        predictorMap.get(token).add(nextToken);
+      }
+      else{
+        // if not, we need to put the next token into the list 
+        // so that it can match the data type we need to put it into the hashmap
+        // and later add more nextWord tokens into the array
+        // then we put this in as a new entry to the hashMap 
+        List<String> nextTokenList = new ArrayList<>();
+        nextTokenList.add(nextToken);
+        predictorMap.put(token, nextTokenList);
+      }
+    }
+
+
+
     // TODO: Convert the trainingWords into neighborMap here
+
+    // Not sure where this is getting pull for later although I am just literally
+    // taking the todo and assigning it to my product map 
+    neighborMap = predictorMap;
```

Comment:
`neighborMap` ends up being used in `predictNextWord` and `getNeighborMap`. It's an instance variable, so it can be used later, unlike `predictorMap` which is a local variable. You actually didn't need `predictorMap` as a separate variable in the first place - you could've just directly used `neighborMap`.